source/transfer_list.rst

```diff
@@ -657,5 +658,65 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM info table entry layout (XFERLIST_TPM_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This entry type holds TPM-related information for a platform. The TPM event log
+info is a region containing a TPM event log as defined by TCG EFI Protocol
+Specification (https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification/).
+The CRB info defines the address of a region of memory that has been carved out
+and reserved for use as a TPM Command Response Buffer interface.
+
+.. _tab_tpm_info:
+.. list-table:: TPM info type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - 0x19
+
+   * - evlog_base_address
+     - 0x8
+     - 0x8
+     - The physical address of the region that contains the TPM event log.
+
+   * - crb_base_address
+     - 0x8
+     - 0x10
+     - The base address of CRB.
+
+   * - evlog_size 
+     - 0x4
+     - 0x18
+     - Size of the event log
+
+   * - crb_size 
+     - 0x4
+     - 0x1C
+     - Size of CRB. crb_size == 0 means that crb_base_address is not valid
+
+   * - need_to_replay
```

Comment:
nit: I would consider calling this "flags" and defining bit 0 as the need_to_replay flag, with other bits reserved for future use.

Reply:
Agree this is a good idea.